lib/Support/LoweringOptions.cpp

```diff
@@ -109,9 +110,10 @@ void LoweringOptions::parse(StringRef text, ErrorHandlerT errorHandler) {
       }
     } else if (option.consume_front("wireSpillingHeuristic=")) {
       if (auto heuristic = parseWireSpillingHeuristic(option)) {
-        wireSpillingHeuristic = *heuristic;
+        wireSpillingHeuristicSet |= *heuristic;
```

Comment:
Error message and parsing logic suggest you can set `=spillNone`-- which doesn't have any effect (and maybe is strange if specified in conjunction with the other options, or at all, other than being explicitly about the preferred behavior regardless of defaults).
Not big deal, but maybe it should clear/override existing options or not  be allowed to be specified to keep these constructive?

Reply:
Good points, probably we can just remove `spillNone` to make it simple.